pkgs/development/libraries/opencascade-occt/default.nix

```
@@ -0,0 +1,41 @@
+{ stdenv
+, fetchurl
+, cmake
+, tcl
+, tk
+, vtk
+, mesa_glu
+, libXext
+, libXmu
+, libXi
+, doxygen
+}:
+
+let version = "7.3.0p2";
+    commit = "V${builtins.replaceStrings ["."] ["_"] version}";
+
+in stdenv.mkDerivation {
+
+  name = "opencascade-occt-${version}";
+
+  src = fetchurl {
+    name = "occt-${commit}.tar.gz";
+    url = "https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=${commit};sf=tgz";
+    sha256 = "0nc9k1nqpj0n99pr7qkva79irmqhh007dffwghiyzs031zhd7i6w";
+  };
+
+  buildInputs = [ cmake tcl tk vtk mesa_glu libXext libXmu libXi doxygen ];
```

```suggestion
  nativeBuildInputs = [ cmake ];
  buildInputs = [ tcl tk vtk mesa_glu libXext libXmu libXi doxygen ];
```
cmake is usually only needed at compile time.